pkg/server/describe.go

```diff
@@ -69,11 +69,19 @@ func (srv *Server) describeHandler(w http.ResponseWriter, r *http.Request, log *
 ----------
 `
 
-	tgw.Write([]byte(header))
+	_, err = tgw.Write([]byte(header))
+	if err != nil {
+		log.Errorw("write error", "err", err)
```

Comment:
Can you make it more descriptive? e.g. "which write?"

Reply:
I am not very familiar with the zap logger, but in development, every ERROR, WARN and FATAL show a stack trace, so you know exactly the line where all ERRORs are logged from. If the stack trace is not enabled once we run this in prod, then we should enable caller location (file:line) on all log line, and again we would know where a given log message is coming from.

You could argue that when we do that, we don't even have to add the verbose `write error` which doesn't add much value, but I think it makes it easier to search once you see the error coming up in log files. Still with (file:line) granularite `log.Error(err.Error())` might also be just enough information.

I hope this makes sense.

Reply:
Nevertheless I see no harm in this being more descriptive, so - changed!